governance/polls/Atlas Edit Monthly Cycle Proposal (AEP-9) - April 14, 2025.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Atlas Edit Monthly Cycle Proposal (AEP-9) - April 14, 2025
+summary: This Atlas Edit Proposal aims to implement a staking mechanism for Sky holders by directing a fixed portion of surplus funds to staked tokens on a weekly basis.
+discussion_link: https://forum.sky.money/t/aep-9-begin-returning-income-to-sky-holders-via-staking/26177
+parameters:
+    input_format:
+        type: single-choice
+        abstain: [0]
+    victory_conditions:
+        - {
+            type: 'and',
+            conditions: [
+                { type : plurality },
+                { type : comparison, comparator : '>=', value: 10000 }
+            ]
+        }
+        - {type : default, value : 2 }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2025-04-14T16:00:00
+end_date: 2025-04-28T16:00:00
+---
+# Atlas Edit Monthly Cycle Proposal (AEP-9) - April 14, 2025
+
+The Governance Facilitators have placed an [Atlas Edit Monthly Cycle Proposal](https://sky-atlas.powerhouse.io/#A.1.10.2_Atlas_Edit_Monthly_Cycle-430185a5-fa5d-4664-89cf-21c9e6cfc109|0db3326e) into the [voting system](https://vote.makerdao.com/polling). This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.2_Atlas_Edit_Weekly_Cycle-4a8ad9ad-5c5d-4994-9b46-f04c0e61ce59%7C0db30308) will be active for fourteen days beginning on Monday, April 14 at 16:00 UTC.
```

Comment:
```suggestion
The Governance Facilitators have placed an [Atlas Edit Monthly Cycle Proposal](https://sky-atlas.powerhouse.io/A.1.11.2_Atlas_Edit_Monthly_Cycle/430185a5-fa5d-4664-89cf-21c9e6cfc109%7C0db3326e) into the [voting system](https://vote.makerdao.com/polling). This Governance Poll will be active for fourteen days beginning on Monday, April 14 at 16:00 UTC.
```

I've removed the "poll" link as all linkable references apply only for the weekly cycle :(